elki/src/main/java/ch/ethz/globis/pht/util/Bits.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2011-2015 ETH Zurich. All Rights Reserved.
+ *
+ * This software is the proprietary information of ETH Zurich.
+ * Use is subject to license terms.
+ */
+package ch.ethz.globis.pht.util;
+
+
+/**
+ * Bit-stream manipulation functions.
+ * 
+ * @author ztilmann
+ *
+ */
+public class Bits extends BitsLong {
+
```

Comment:
Yes. My internal working copy of the PhTree has different versions with different bit depths (long, int, byte). Using the above approach it is easy to switch between the representations when creating a copy of the tree.
I can remove it if there are considerable concerns, but it would make porting of future features somewhat more effort.